test/autorest/generated/stringgroup/enums.go

```diff
@@ -10,12 +10,11 @@ import azinternal "generatortests/autorest/generated/stringgroup/internal/string
 type Colors = azinternal.Colors
 
 const (
-	ColorsRedColor = azinternal.ColorsRedColor
-	ColorsGreenColor = azinternal.ColorsGreenColor
-	ColorsBlueColor = azinternal.ColorsBlueColor
+	ColorsRedcolor   = azinternal.ColorsRedcolor
```

Comment:
Unclear what caused this change, regression in the modeler?

Reply:
It's due to the following line in the namer.
```ts
details.name = `${enm.language.go?.name}${capitalizeAcronyms(pascalCase(details.name.toLowerCase()))}`;
```
Not sure why this is showing up now but I think we should remove the call to `toLowerCase()`.  What do you think?

Reply:
Yup agreed! Will you remove it? Or should I make a note of it?

Reply:
I just pushed the fix.

Reply:
Now I'm just not sure it all of the groups need to be regenerated. I saw the the generated output for urlgroup still has color as lower case.